lib/registries/addon/components/registration-form-view/styles.scss

```diff
@@ -19,3 +19,105 @@
 .Answer > p {
     font-size: 16px;
 }
+
+.NavigationMenu {
+    position: absolute;
+    z-index: 100;
+    top: 22px;
+    right: 30px;
+    width: 200px;
+    box-shadow: 0 1px 2px $primary-box-shadow;
+}
+
+.Header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    border-bottom: 1px solid $alto;
+}
+
+.HeaderText {
+    padding-left: 15px;
+    flex-basis: 80%;
+    background-color: $white;
+    font-size: 15px;
+    font-weight: 700;
+    line-height: 2.5;
+}
+
+.ToggleNav__button:global(.btn.btn-link) {
+    float: right;
+    color: $primary-blue;
+    padding: 3px 8px;
+
+    &:hover {
+        color: $secondary-blue;
+    }
+
+    &:focus {
+        outline: 0;
+    }
+
+    i {
+        font-size: 1.1rem;
+    }
+}
+
+.NavigationMenu.Collapsed {
+    box-shadow: none;
+
+    .Header {
+        border-bottom: 0;
+        justify-content: flex-end;
+
+        .ToggleNav__button {
+            border: 1px solid $alto;
+            border-radius: 2px;
+            background-color: $white;
+            background-size: cover;
+        }
+
+        .ToggleNav__button:hover {
+            background-color: $alto;
+        }
+    }
+}
+
+.Anchors {
+    height: 300px;
+    overflow: scroll;
+    padding: 0 15px 15px;
+    background-color: $white;
+
+    a {
```

Comment:
For long titles, it's hard to tell where one starts and the next begins. Maybe add a `padding-bottom: 5px` here?